civictechprojects/static/css/partials/_LogInController.scss

```
@@ -65,15 +65,43 @@
 }
 
 .LogInController-socialLink {
-  margin: 0 10px;
   background-color: transparent;
   text-decoration: none;
+  border: thin solid;
```

```suggestion
  border: 1px solid $color-background-dark;
```
border thin isn't guaranteed to be cross-browser consistent, so this is a minor "just in case" fix.